kpatch-build/kpatch-build

```diff
@@ -601,7 +604,8 @@ cp "$OBJDIR/.config" "$SRCDIR"
 cd "$SRCDIR"
 make prepare >> "$LOGFILE" 2>&1 || die
 
-if [[ $DISTRO == ubuntu ]]; then
+# UTS_UBUNTU_RELEASE_ABI: is added only, if the kernel is not a stock kernel.
```

Comment:
I actually think this comment is confusing, because I interpret "stock kernel" to mean "the kernel which comes with Ubuntu by default", whereas I think you really mean "the upstream kernel".  And actually I don't think either of the comments in this patch are necessary, so if you just remove all the comments, I'll give it a 👍 :-)